example/example/templates/middleware-tester-table.html

```diff
@@ -57,7 +57,7 @@
     <tr>
       <td>
         <code>request.htmx.triggering_event</code> <br>
-        <small>(via <a href="https://htmx.org/extensions/event-header/">event-header extension</a>)</small>
+        <small>(via <a href="https://v1.htmx.org/extensions/event-header/">event-header extension</a>)</small>
```

Comment:
example project is using v2 htmx so let's always link there

```suggestion
        <small>(via <a href="https://github.com/bigskysoftware/htmx-extensions/blob/main/src/event-header/README.md">event-header extension</a>)</small>
```